pontoon/messaging/templates/messaging/includes/compose.html

```diff
@@ -0,0 +1,224 @@
+                </div>
+                <div class="field clearfix to half">
+                    {{ form.review_to.label_tag() }}
+                    {{ form.review_to }}
+                </div>
+            </div>
+        </section>
+
+        <section class="last-login clearfix">
+            <h3>Filter by last login</h3>
+            <div class="date">
+
+                <div class="field clearfix from half">
+                    {{ form.login_from.label_tag() }}
+                    {{ form.login_from }}
+                </div>
+                <div class="field clearfix to half">
+                    {{ form.login_to.label_tag() }}
+                    {{ form.login_to }}
+                </div>
+            </div>
+        </section>
+    </form>
+    <menu class="controls">
+        <button class="button toggle review" data-target="#review">Review message<span
+                class="fa fa-chevron-right"></span></button>
+    </menu>
+</div>
+<div id="review">
+    <section class="message-content clearfix">
+        <h3>Review message</h3>
+        <div class="content-wrapper">
+            <div class="subject">
+                <p class="value"></p>
+            </div>
+            <div class="body">
+                <div class="value"></div>
+            </div>
+        </div>
+    </section>
+    <section class="recipients clearfix">
+        <h3>Recipients</h3>
+        <div class="user-roles">
+            <h5>User roles</h5>
+            <p class="value"></p>
+        </div>
+        <div class="locales">
+            <h5>Locales</h5>
+            <p class="value"></p>
+        </div>
+        <div class="projects">
+            <h5>Projects</h5>
+            <p class="value"></p>
+        </div>
+        <div class="submitted-translations">
+            <h5>Submitted translations</h5>
+            <p class="value amount"></p>
+            <p class="value date"></p>
+        </div>
+        <div class="performed-reviews">
+            <h5>Performed reviews</h5>
+            <p class="value amount"></p>
+            <p class="value date"></p>
+        </div>
+        <div class="last-login">
+            <h5>Last login</h5>
+            <p class="value date"></p>
+        </div>
+    </section>
+    <section class="message-type clearfix">
+        <h3>Message type</h3>
+        <p class="value"></p>
+        <p class="transactional"><i>Warning:</i> Transactional emails are also sent to users who have
+            not opted in to email communication. They are restricted in the type of content that can be
+            included. When in doubt, please review with legal.</p>
+    </section>
+    <menu class="controls">
+        <button class="button toggle edit" data-target="#compose"><span class="fa fa-chevron-left"></span>Back to
+            editing</button>
+        <div class="right">
+            <button class="button send to-myself">Send to myself</button>
+            <button class="button active send">Send</button>
```

Comment:
It's not clear that "Send to myself" will not save the message as "Sent", like "Send" does.

Will a later update be introducing something like "Drafts" for messages that have not been sent?

Reply:
I overlooked that the spec actually says "Send **test** to myself". That should make it clearer.

We'll see if there's actually a need for something like Drafts and the prioritize accordingly. Even showing Sent messages and adding the ability to recycle them wasn't initially planned.